packages/dx-react-grid-bootstrap3/src/templates/table-fixed-cell.jsx

```diff
@@ -1,8 +1,21 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
 import { ThemeColors } from './layout';
+import { getStickyPosition } from '../utils/css-fallback-properties';
 
 export class FixedCell extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stickyPosition: getStickyPosition(),
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ stickyPosition: getStickyPosition() });
```

Comment:
We need to avoid re-render here when stickyPosition hasn't changed - like in Table component

Reply:
@ushkal  It is a Pure component